src/RestController.php

```diff
@@ -36,7 +36,7 @@ public function __construct($post_type)
     /**
      * {@inheritDoc}
      */
-    public function get_item_permissions_check($request)
+    public function get_item_permissions_check($request) // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
```

Comment:
Why you will not work with our codestyle there based on PSR, without _ in the function name? A ignore for PHPCS is not a solution in this context. Thanks for change them.

Reply:
Because it needs to override the inherited method.

Reply:
Thanks for pointing this, I haven't seen this is a core function.